dask/array/linalg.py

```diff
@@ -829,22 +842,39 @@ def svd(a):
             "Input ndim: {}\n".format(a.shape, a.ndim)
         )
     if nb[0] > 1 and nb[1] > 1:
-        raise ValueError(
+        raise NotImplementedError(
             "Array must be chunked in one dimension only. "
             "This function (svd) only supports tall-and-skinny or short-and-fat "
             "matrices (see da.linalg.svd_compressed for SVD on fully chunked arrays).\n"
             "Input shape: {}\n"
             "Input numblocks: {}\n".format(a.shape, nb)
         )
 
-    # Tall-and-skinny case
-    if nb[0] >= nb[1]:
-        u, s, v = tsqr(a, compute_svd=True)
-        return u, s, v
-    # Short-and-fat case
+    # Single-chunk case
+    if nb[0] == nb[1] == 1:
+        u, s, v = map(asarray, np.linalg.svd(np.asarray(a), full_matrices=False))
```

Comment:
This will trigger computation.   I think that instead we probably want to keep things lazy, and instead return three single-chunked dask arrays.  Thoughts?

Reply:
I gave it a shot with https://github.com/dask/dask/pull/6616/commits/2d7194f47681e7f1be297f0aac7c15887c4bee4f#diff-d190ab15173165ce6615a5df4e3067ebR862-R868.  That look ok?